.github/workflows/bikes.yml

```diff
@@ -13,7 +13,14 @@ jobs:
     steps:
     
     - uses: actions/checkout@master
-    
+
+    - name: Create valid child namespace name
+      id: replace
```

Comment:
This is a pretty generic id. I would pick something specific to the step, like `generate-child-space-name` and then name the output like `result`, so the way of referencing it later is `${{steps.generate-child-space-name.result}}`.

Reply:
updated